src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -7637,7 +7637,7 @@ private static IEnumerable<JsonWriterOptions> JsonOptions()
             return from indented in new[] { true, false }
                    from skipValidation in new[] { true, false }
                    from indentCharacter in indented ? new char?[] { null, ' ', '\t' } : []
-                   from indentSize in indented ? new int?[] { null, 0, 1, 2, 127 } : []
```

Comment:
Do we still have coverage of the `127` edge case?

Reply:
Looks like we do https://github.com/dotnet/runtime/blame/b229723b2b110f3e948c65c3065fb48da27fab6d/src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs#L6787.